submission/submission-webapp/src/itest/kotlin/ac/uk/ebi/biostd/itest/factory/AllInOne.kt

```diff
@@ -21,5 +21,4 @@ fun submissionsFiles(tempFolder: File): List<SubmissionFile> = listOf(
     SubmissionFile(tempFolder.createOrReplaceFile("DataFile5.txt", "content 5")),
     SubmissionFile(tempFolder.createOrReplaceFile("DataFile6.txt", "content 6"), "Folder1"),
     SubmissionFile(tempFolder.createOrReplaceFile("DataFile7.txt", "content 7")),
-    SubmissionFile(tempFolder.createOrReplaceFile("DataFile8.txt", "content 8"), "Folder1"),
```

Comment:
what is the reason of this change?

Reply:
The only reason for this file was to have a second file to reference in the file list test. Now that I included the folder itself as the second reference, I thought it no longer makes sense to have it